milli/Cargo.toml

```diff
@@ -137,3 +137,6 @@ greek = ["charabia/greek"]
 
 # allow khmer specialized tokenization
 khmer = ["charabia/khmer"]
+
+# allow CUDA support
```

Comment:
Can you link this PR to this feature flag or find another way to link to the documentation you provided, please?